### DISCUSSION TITLE
第1刷（2022年8月1日発行）の誤字・脱字について

### DISCUSSION THREAD

**Discussion by magavel** [budougumi0617/go_todo_app#66] [Ideas]
### p.172 SECTION-067 8行目
`リクエストの処理が正常が完了する場合`
↓
`リクエストの処理が正常に完了する場合`でしょうか？

### p.218 SECTION-079 リスト19.23 2行目
`// ErrCodeMySQLDuplicateEntry はMySQL系ののDUPLICATEエラーコード`
↓
`// ErrCodeMySQLDuplicateEntry はMySQL系のDUPLICATEエラーコード`でしょうか？

### p.226 SECTION-082 9行目
`Redisの接続情をの差分を吸収するため`
↓
`Redisの接続情報の差分を吸収するため`でしょうか？

### p.236 SECTION-084 2〜3行目
`OSSを利用してしてダミーデータを生成`
↓
`OSSを利用してダミーデータを生成`でしょうか？

### p.257 SECTION-087 5行目
`アクセストークンからユーザーIDをできるようになった`
↓
`アクセストークンからユーザーIDを取得できるようになった`でしょうか？

※第2刷ですでに修正されていましたらすみません！

**Comment by budougumi0617** [comment]
正誤表に反映しました。ご指摘ありがとうございました！
https://github.com/budougumi0617/go_todo_app/releases/tag/v1.0.8
